aten/src/ATen/native/cpu/CopyKernel.cpp

```diff
@@ -29,6 +29,12 @@ static void copy_kernel(TensorIterator& iter, bool non_blocking) {
   if (dtype == iter.dtype(1)) {
     if (dtype == ScalarType::Half) {
       unary_kernel(iter, [=](at::Half a) -> at::Half { return a; });
+    } else if (isQIntType(dtype)) {
```

Comment:
is the reason you can't use AT_DISPATCH_ALL_TYPES_AND2 below that vectorized things are not defined for qtypes? might be worth to define them - but if it's too hard, I'm fine doing it in a separate diff

Reply:
Yes. sure I can try that in another diff